integrations/LaunchDarkly/utils.js

```diff
@@ -1,6 +1,6 @@
-const createUser = (message) => {
+const createUser = (message, keyOverride = undefined) => {
   const user = {};
-  user.key = message.userId || message.anonymousId;
+  user.key = keyOverride || message.userId || message.anonymousId;
```

Comment:
Should we keep the priority of keyOverride before `userId` in that case even for identified users where `userId` is present we will still consider the overriding key.

should we keep the priority order like this
```
userId || keyOverride || anonymousId
```

Reply:
Yes, I'm going to push a change that will only apply the `keyOverride` if you're anonymous.